loom/engine/loom2d/l2dRectangle.h

```diff
@@ -260,15 +260,18 @@ class Rectangle
     /**
      * Make a copy of this Rectangle.
      */
-    Rectangle *clone()
+    int clone(lua_State *L)
```

Comment:
Cool, can you add a card to check for other places where we might do something similar? E.g. it's possible that the vector code that I wrote does it.